cmd/lnd/main.go

```diff
@@ -9,12 +9,20 @@ import (
 )
 
 func main() {
+	// Load the configuration, and parse any command line options. This
+	// function will also set up logging properly.
+	loadedConfig, err := lnd.LoadConfig()
```

Comment:
Nice! Yeah this is a key move to allow the callers to eventually manually populate things like the backing database instance themselves manually.